dist/tools/travis-scripts/build_and_test.sh

```diff
@@ -45,6 +45,15 @@ then
         ./dist/tools/externc/check.sh master
         RESULT=$(set_result $? $RESULT)
 
+       ./dist/tools/endingcheck/check.sh master --diff-filter=MR
+       RESULT=$(set_result $? $RESULT)
+
+       ./dist/tools/endingcheck/check.sh master --diff-filter=AC
+       RESULT=$(set_result $? $RESULT)
+
+       ./dist/tools/endingcheck/check.sh riot/master
```

Comment:
One call of your script with `riot/master` (the current master at RIOT-OS/RIOT) and `--diff-filter=MA` (only look at modified [M] and added [A] files) should suffice (source: http://git-scm.com/docs/git-diff).